superchain/configs/sepolia/race.yaml

```diff
@@ -0,0 +1,18 @@
+name: race
+chain_id: 90001
+public_rpc: https://racetestnet.io
+sequencer_rpc: https://racetestnet.io/sequencer
+explorer: https://testnet.racescan.io
+
+superchain_level: 2
+
+batch_inbox_addr: "0xff00000000000000000000000000000000042069"
+
+genesis:
+  l1:
+    hash: "0x4adb153f934e816f659868e215e5b28c2759920cb1dc26f261c17a22cab36506"
+    number: 5628480
+  l2:
+    hash: "0xb76c30ebd182ec16bbf711fa8c8fa6adb569604bd36603ba1fece9d3a564d09b"
```

Comment:
Hi [@nitantchhajed](https://github.com/nitantchhajed) we have been digging into this  for you today, and have some updates that should help us move towards getting that last check passing.

The root cause of the failing check is that we re-compute your l2 genesis block hash from the configuration information you have supplied on this PR (generated by the add-chain.sh script we asked you to run), and it doesn't match your actual l2 genesis block hash.

We have a bug in our add-chain.sh script, which we are fixing here https://github.com/ethereum-optimism/superchain-registry/pull/165. In the case where your ecotone_time is different than the superchain wide value specified here superchain/configs/sepolia/superchain.yaml, we would be incorrectly assume you were using the latter value.

Whether you have ecotone activated at genesis or not changes whether cancun is activated or not, and this affects the l2 genesis block hash https://github.com/ethereum-optimism/op-geth/blob/6aa2348118138a4f7021208ffaf8582d73169ba1/core/genesis.go#L534-L549 , so the above bug could lead to a false negative in our checks.

However, I did some experimentation, and coupled with your comments above it seems you generated a genesis without ecotone enabled, and then retroactively tried to activate ecotone at genesis with a CLI flag to avoid op-geth panicking. Unfortunately this is not a valid way to activate ecotone at genesis.

Can I ask you to try one more time to restart your chain with delta_time and ecotone_time set to 0 in the `rollup.json` and `genesis.json` and ensure that you reinitialize geth before starting up? If you still hit a problem, please share as much information as possible to help us continue to debug this with you.

Reply:
hii @geoknee  I added `"deltaTime":0 "cancunTime":0 "ecotoneTime":0` in `genesis.json` & added `ecotone_time:0  delta_time:0`  in `rollup.json`.
re-initialized the `op-geth` and started the chain.

logs form `op-node`:

```
t=2024-04-05T19:50:53+0000 lvl=info msg="Not opted in to ProtocolVersions signal loading, disabling ProtocolVersions contract now."
t=2024-04-05T19:50:53+0000 lvl=info msg="No persisted sequencer state loaded"
t=2024-04-05T19:50:53+0000 lvl=info msg="Rollup Config" l2_chain_id=90001 l2_network="unknown L2" l1_chain_id=11155111 l1_network=sepolia l2_start_time=1712246880 l2_block_hash=0xb76c30ebd182ec16bbf711fa8c8fa6adb569604bd36603ba1fece9d3a564d09b l2_block_number=0 l1_block_hash=0x4adb153f934e816f659868e215e5b28c2759920cb1dc26f261c17a22cab36506 l1_block_number=5628480 regolith_time="@ genesis" canyon_time="@ genesis" delta_time="@ genesis" ecotone_time="@ genesis" fjord_time="(not configured)" interop_time="(not configured)"
t=2024-04-05T19:50:53+0000 lvl=info msg="Initializing rollup node" version=v0.0.0-99a53381-1710933867
t=2024-04-05T19:50:54+0000 lvl=info msg="Connected to L1 Beacon API, ready for EIP-4844 blobs retrieval." version=Lighthouse/v5.1.3-3058b96/x86_64-linux
t=2024-04-05T19:50:54+0000 lvl=eror msg="Error initializing the rollup node" err="failed to init L2: failed to get L2 genesis blockhash: expected L2 genesis hash to match L2 block at genesis block number 0: 0x7d0e1839d7b0489a41ba6f1f9c320fbb459c24114ad6f459cca5dbe39f4373bb <> 0xb76c30ebd182ec16bbf711fa8c8fa6adb569604bd36603ba1fece9d3a564d09b"
t=2024-04-05T19:50:54+0000 lvl=crit msg="Application failed" message="failed to setup: unable to create the rollup node: failed to init L2: failed to get L2 genesis blockhash: expected L2 genesis hash to match L2 block at genesis block number 0: 0x7d0e1839d7b0489a41ba6f1f9c320fbb459c24114ad6f459cca5dbe39f4373bb <> 0xb76c30ebd182ec16bbf711fa8c8fa6adb569604bd36603ba1fece9d3a564d09b"
```

Reply:
I even tried to create the completely different `rollup.json` and `genesis.json` files for a separate chain with some different chain ID and new set of contract deployments also. But still getting this Hash mismatch issue when I put the fork timestamps in rollup and genesis files. What could be the issue ? @sbvegan @geoknee

Reply:
I think you need to add these parameters in the `deploy-config` getting-started scripts so that new superchains will not face the problems of -
1. Ecotone.
2. Genesis Hash mismatch

Reply:
@zchn tracking that here https://github.com/ethereum-optimism/security-pod/issues/107